### PR TITLE
Fix event stream.

### PR DESCRIPTION
**Pull request by nkoenig**
Fix an error in the event stream, and also improve the artifact report events

The matches the event stream modification in pull request #545.

Signed-off-by: Nate Koenig <nate@openrobotics.org>

### CODE REVIEW

File: subt_ign/include/subt_ign/CommonTypes.hh [hunk 1]
```diff
@@ -66,8 +66,8 @@ namespace subt
   enum class ArtifactType : uint32_t
   {
     TYPE_BACKPACK = 0,
-    TYPE_DUCT,
     TYPE_DRILL,
+    TYPE_DUCT,
```
Comment (angelacmaio) [inline subt_ign/include/subt_ign/CommonTypes.hh:70]: Was this change intentional?

Comment (nkoenig) [reply subt_ign/include/subt_ign/CommonTypes.hh:70]: Yes, so that the indices match this: https://github.com/osrf/subt/blob/master/subt_ign/src/GameLogicPlugin.cc#L78
